src/public/auth.html

```diff
@@ -202,7 +202,15 @@ <h1 class="pb-4">Azure Static Web Apps Auth</h1>
           const urlSearch = (metaSearch || location.search).replace("?", "");
           const urlQuery = urlSearch && Object.fromEntries(new Map(urlSearch.split("&").map((query) => query.split("="))));
           const postLoginRedirectUri = urlQuery ? urlQuery["post_login_redirect_uri"] : redirectPath;
-          window.location.href = postLoginRedirectUri || defaultRedirectPath;
+          const decodeRef = urlQuery && urlQuery["decode_redirect"];
```

Comment:
I am not sure we need this `decode_redirect` flag. I think we can safely call `decodeURIComponent(postLoginRedirectUri)` by default this will try to decode the URI even if it wasn't encoded:

```javascript
encodeURIComponent('https://github.com/Azure/static-web-apps-cli/pull/543/files')
// 'https%3A%2F%2Fgithub.com%2FAzure%2Fstatic-web-apps-cli%2Fpull%2F543%2Ffiles'

decodeURIComponent('https%3A%2F%2Fgithub.com%2FAzure%2Fstatic-web-apps-cli%2Fpull%2F543%2Ffiles');
// 'https://github.com/Azure/static-web-apps-cli/pull/543/files'

decodeURIComponent('https://github.com/Azure/static-web-apps-cli/pull/543/files');
// 'https://github.com/Azure/static-web-apps-cli/pull/543/files'


```